### ISSUE TITLE
Encourage proper capitalisation of users' names

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#6828]
I think properly capitalised names help make services, and correspondence, look smart. 

The system could ask those who enter their names in all lower case on sign up if that's what they intend. 

Some users may well be choosing all lowercase names as some kind of style statement perhaps, perhaps seeking to invoke informality?  

See also:

* https://github.com/mysociety/whatdotheyknow-theme/issues/787
* https://github.com/mysociety/alaveteli/issues/6154

**Comment by RichardTaylor** [comment]
We have a user today who is struggling to use the service who has capitalised their first name but not their surname. 

We should catch such improperly capitalised names as well as all lowercase names if we were to offer a prompt. 

It may be that there is a correlation between those who don't properly capitalise their names and those with a low level of literacy and those who struggle using the service and using FOI.

**Comment by garethrees** [comment]
Related to users signing up with a single name https://github.com/mysociety/alaveteli/issues/6284.

**Comment by RichardTaylor** [comment]
A couple of those struggling to use WhatDoTheyKnow and FOI have capitalised two letters at the start of a name rather than just one. This should also be caught by any system for offering guidance. 

I suspect this kind of occurrence might be as a result of a lack of typing skills, or struggling to physically type.

**Comment by HelenWDTK** [comment]
This issue has been automatically closed due to a lack of discussion or resolution for over 12 months.
Should we decide to revisit this issue in the future, it can be reopened.
